README.md

```diff
@@ -16,20 +16,31 @@ with [Anthos Service Mesh] features enabled.
     ```bash
     # [PACKAGE] is any directory name under the repository root.
     # [OUTPUT_DIR] refers to the desired local location of the package.
-    $ kpt pkg get
-    https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/[PACKAGE] [OUTPUT_DIR]
+    $ kpt pkg get https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/[PACKAGE] [OUTPUT_DIR]
+
+    # e.g. to get the asm package into the current folder, run the following,
+    # which will create an 'asm' folder under the current folder
+    $ kpt pkg get https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/asm .
     ```
 4.  Verify all configurable values are as desired:
 
     ```bash
-    $ kpt cfg list-setters [OUTPUT_DIR]
+    $ kpt cfg list-setters [ASM_FOLDER]
+    # Note that by default, 'kpt get' will have populated the project and the compute zone
+    # to match your current configuration.
+
     # If any values aren't what you want, run something like below:
-    $ kpt cfg set [OUTPUT_DIR] [NAME] [VALUE]
+    $ kpt cfg set [ASM_FOLDER] [NAME] [VALUE]
+
+    # e.g. you can change the cluster and compute zone as follows
+    $ kpt cfg set asm gcloud.core.project my-project-id
```

Comment:
use [ASM_FOLDER] here and below

Reply:
This one is a complete example, so it's deliberate to use the same concrete folder from the previous example.